### PR TITLE
Docs: Reference new feature guidelines and API tenets

### PR DESCRIPTION
**Pull request by gibson042**
Fixes gh-2320

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,9 +12,11 @@ Note: This is the code development repository for *jQuery Core* only. Before ope
 
 ## Getting Involved
 
-We've put together [a short guide with tips, tricks, and ideas on getting started](http://contribute.jquery.org/open-source/). We're always looking for help identifying bugs, writing and reducing test cases, and documentation.
+[jQuery tenets](https://github.com/jquery/jquery/wiki/API-design-guidelines)
```
Comment (mzgol) [inline CONTRIBUTING.md]: Is `tenet` a common word? I haven't heard it before so it might a barrier to non-native speakers.

Comment (markelog) [inline CONTRIBUTING.md]: +1

Comment (markelog) [inline CONTRIBUTING.md]: `Principles` might be better?

**Comment by dmethvin** [comment]
LGTM now.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,9 +12,11 @@ Note: This is the code development repository for *jQuery Core* only. Before ope
 
 ## Getting Involved
 
-We've put together [a short guide with tips, tricks, and ideas on getting started](http://contribute.jquery.org/open-source/). We're always looking for help identifying bugs, writing and reducing test cases, and documentation.
+[API design principles](https://github.com/jquery/jquery/wiki/API-design-guidelines)
```
Comment (markelog) [inline CONTRIBUTING.md]: Maybe we should change the path for it too? `API-design-guidelines` -> `API-design-principles`

**Pull request closed without merging**